### DISCUSSION TITLE
Ascalon Burrows

### DISCUSSION THREAD

**Discussion by RealAredian** [blish-hud/Blish-HUD#669] [Requested Marker Packs]
Hi, would you maybe be able to create number markers for the burrows that appear in the Ascalonian Catacomb Dungeon on Path 3? Would be super helpful! 

Like in this picture for example:
![1920px-Sequence_burrow](https://user-images.githubusercontent.com/103974596/163902219-f6038366-cceb-4cce-9fc2-925f4948f22c.jpg)

**Comment by Metallis** [comment]
Are these not already included in Tekkit's, Reactif's, or other packs?

**Reply by RealAredian** [reply]
Unfortunately not, I have all available marker packs installed and none of them include the burrows.

**Reply by Metallis** [reply]
This is actually covered as part of Hero's Timers (download within the Timers module). https://github.com/QuitarHero/Hero-Timers/wiki#-ascalonian-catacombs-p3-gravelings but may also be enabled within Hero's marker pack at some point. https://github.com/QuitarHero/Heros-Marker-Pack/wiki
